src/ios/CDVLocation.h

```diff
@@ -66,5 +68,6 @@ typedef NSUInteger CDVLocationStatus;
 - (void)locationManager:(CLLocationManager*)manager
        didFailWithError:(NSError*)error;
 
-- (BOOL)isLocationServicesEnabled;
+- (void)isLocationServicesEnabledWithCompletion:(void (^)(BOOL enabled))completion;
+//- (BOOL)isLocationServicesEnabled;
```

Comment:
This comment is not needed, you can remove it.

Reply:
Done.